packages/web/src/components/navbar/Navbar.tsx

```diff
@@ -11,32 +11,22 @@ import {
   Profile as ProfileIcon,
   Logout as LogoutIcon,
 } from '@components/icons';
-import React, { useEffect, useRef, useState } from 'react';
+import React, {useRef, useState } from 'react';
 import { Button } from '@components/common/Button';
 import { NavItem } from '@components/navbar/NavItem';
 import { DropdownItem } from '@components/common/DropdownItem';
 import { useGetCurrentUser } from '@lib/common/getCurrentUser';
 import { useRouter } from 'next/router';
+import useOnClickOutside from 'src/hooks/useOnClickOutside';
```

Comment:
Avoid relative paths. We use absolute paths, change this to "@hooks/useOnClickOutside", and in tsconfig.json follow the other absolute paths format to add @hooks/.